cypress/e2e/hub/collections.cy.ts

```diff
@@ -31,7 +30,8 @@ describe.skip('Collections- List View', () => {
         cy.get('td[data-cy=checkbox-column-cell]').click();
       });
       cy.get('[data-cy="Submit"]').click();
-      cy.clickButton(/^Clear all filters$/);
+      cy.url().should('include', 'approvals');
+      cy.reload();
```

Comment:
I was able to remove this `cy.reload` and tests passed locally. I did not get why we need that one.